zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -215,6 +218,19 @@ def __init__(self):
         self.http_server.register_async_uri(self.DELETE_SNAPSHOT_PATH, self.delete_snapshot)
         self.http_server.register_async_uri(self.ROLLBACK_SNAPSHOT_PATH, self.rollback_snapshot)
 
+    @replyerror
+    def get_facts(self, req):
```

Ruff (0.12.2)

Unused method argument: `req`

(ARG002)